index.html

```diff
@@ -1,30 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
-
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    <nav id="navbar">
-      <a href="#projects">Projects</a>
-      <a href="https://github.com/Mucyosoda/claude-taibu" id="profile-link" target="_blank">Profile</a>
-    </nav>
-    <main>
-      <section id="welcome-section">
-        <h1>Personal Portfolio</h1>
-        
-      </section>
-      <section id="projects">
-        <div class="project-tile"></div>
-        <a href="">project1</a>
-      </section>
-    </main>
-    <script  src="js/index.js"></script>
```

Comment:
Please should avoid spaces in your code to make the code look professional, I don't see why you add this line.

- [ ] you can try to put <script> at the end of code before </body> if you want!